unfoldingWord/translationCore
# Add version to welcome and splash

#### This pull request addresses:

fixes Issue #1454 and the fail on #963 



#### How to test this pull request:

Confirm version number shows up on splash screen and welcome page


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1467)
